pkgs/tools/misc/killport/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchCrate
+, llvmPackages
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "killport";
+  version = "0.7.0";
```

```suggestion
  version = "0.8.0";
```

v0.8.0 released in the meantime
https://github.com/jkfran/killport/releases/tag/v0.8.0

(don't forget to update the hashes below too)

---

I updated and ready to merge(if check passed)